tests/release.yaml

```diff
@@ -0,0 +1,10 @@
+# Contains all operators required to run the test suite.
```

Comment:
in the test we use secrets and a s3 bucket so we need secret/commons here too.

Reply:
Done: https://github.com/stackabletech/edc-operator/pull/58/commits/26fc8920b9a1756cb127ff8eed23822dc8e2f874